src/lib/Scenes/MyProfile/MyCollectionAndSavedWorks.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { ViewProps } from "react-native"
+
+import { MyProfile_me } from "__generated__/MyProfile_me.graphql"
+import { FancyModalHeader } from "lib/Components/FancyModal/FancyModalHeader"
+import { StickyTabPage } from "lib/Components/StickyTabPage/StickyTabPage"
+import { navigate } from "lib/navigation/navigate"
+import { Box, Sans, SettingsIcon as _SettingsIcon, useColor, useSpace } from "palette"
+import { FavoriteArtworksQueryRenderer } from "../Favorites/FavoriteArtworks"
+import { MyCollectionQueryRenderer } from "../MyCollection/MyCollection"
+
+export enum Tab {
+  collections = "Collections",
```

Comment:
```suggestion
  collections = "Collection",
```
I think this copy should be `Collection`